includes/data-port/export-tasks/class-sensei-export-task.php

```diff
@@ -104,12 +104,18 @@ public function __construct( Sensei_Data_Port_Job $job ) {
 
 	/**
 	 * Run export task.
+	 *
+	 * @throws Exception Caught exception when empty file is passed in tests.
 	 */
 	public function run() {
 		$posts = $this->query->posts;
 		$job   = $this->get_job();
 
 		try {
+			if ( empty( $this->file ) ) {
+				throw new Exception( 'Missing output file' );
```

Comment:
I didn't translate this because it is never actually used. We don't use the Exception message, just catch it.